src/model/wallet.ts

```diff
@@ -211,7 +211,7 @@ const authenticationPrompt = lang.t('wallet.authenticate.please');
 
 export const createdWithBiometricError = 'createdWithBiometricError';
 
-const isHardwareWalletKey = (key: string | null) => {
+const isHardwareWalletPrivateKey = (key: string | null) => {
```

Comment:
Should we consider a better differentiator for this? Like a prefix or something `hardware_${device}_${index}`

Reply:
would rename this as calling this a private key is misleading

Reply:
we should make room to more clearly distinguish between other HW wallet signers

Reply:
i think this should be pretty easy to add with the current flow's will just have to hoist a few routing functions for diff formats, ideally id like to keep this part the same `${deviceid}/${index}` and use `walletType` to then tell us which signer + connection flow to use